### PR TITLE
feat: backport - AUT-1323 allow exporting passages with inline style …

### PR DESCRIPTION
**Pull request by pnal**
Backport of #348

### CODE REVIEW

File: model/export/service/MediaResourcePreparerInterface.php [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA;
```
Comment (bartlomiejmarszal) [inline model/export/service/MediaResourcePreparerInterface.php:18]:
```suggestion
 * Copyright (c) 2022 (original work) Open Assessment Technologies SA;
```

File: scripts/install/RegisterMediaResourcePreparer.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
```
Comment (bartlomiejmarszal) [inline scripts/install/RegisterMediaResourcePreparer.php:17]:
```suggestion
 * Copyright (c) 2022 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
```

File: scripts/install/RegisterMediaResourcePreparer.php [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+        return Report::createSuccess(
+            sprintf(
+                'Successfully registered service \'%s\' on service key: \'%s',
+                $params['service'],
+                MediaResourcePreparerInterface::SERVICE_ID
+            )
+        );
+    }
+}
```
Comment (bartlomiejmarszal) [inline scripts/install/RegisterMediaResourcePreparer.php:43]:
```suggestion
}

```

File: scripts/install/RegisterMediaResourcePreparer.php [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+use oat\taoMediaManager\model\export\service\MediaResourcePreparerInterface;
+
+class RegisterMediaResourcePreparer extends InstallAction
+{
+    public function __invoke($params = []): Report
+    {
+        $mediaResourcePreparer = new $params['service']();
+
+        $serviceManager = $this->getServiceManager();
+        $serviceManager->register(MediaResourcePreparerInterface::SERVICE_ID, $mediaResourcePreparer);
```
Comment (bartlomiejmarszal) [inline scripts/install/RegisterMediaResourcePreparer.php:33]:
```suggestion
        $this->getServiceManager()->register(MediaResourcePreparerInterface::SERVICE_ID, $mediaResourcePreparer);
```

**Review by bartlomiejmarszal — CHANGES_REQUESTED** [review]
**Review Checklist**
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [ ] Pull request's target is not `master`
- [ ] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Commits messages are meaningful
- [ ] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by bartlomiejmarszal — APPROVED** [review]

File: model/export/service/MediaResourcePreparerInterface.php (hunk 1 above)
Comment (pnal) [reply model/export/service/MediaResourcePreparerInterface.php:18]: this is backport, the code from 2021

**Pull request merged**